proxy/http/HttpTunnel.cc

```diff
@@ -1311,7 +1311,8 @@ HttpTunnel::consumer_reenable(HttpTunnelConsumer *c)
           if (is_debug_tag_set("http_tunnel"))
             Debug("http_tunnel", "Unthrottle %p %" PRId64 " / %" PRId64, p, backlog, p->backlog());
           srcp->unthrottle();
-          srcp->read_vio->reenable();
+          if (srcp->read_vio)
```

Comment:
Extremely nitpicky, and I don't feel super strongly, but I think we've been trying to avoid the if()'s without {} going forward.